v1.9/kubeadm/README.md

```diff
@@ -0,0 +1,99 @@
+# kubeadm
+Official documentation:
+ - https://kubernetes.io/docs/setup/independent/install-kubeadm/
+ - https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/
+
+
+# Reproduce Conformance Results:
+
+## pre-reqs
+- vagrant + virtualbox
+
+## install
+The following Vagrantfile will:
+- setup an Ubuntu 17.10 vm with 4 CPUs and 4GB of RAM
+- install a single node k8s cluster with upstream kubeadm, and Weave 1.6
+- start Sonobuoy /w e2e kube-conformance via [sonobuoy-conformance.yaml](../../sonobuoy-conformance.yaml)
+
+The virtualbox provider is assumed.
+```ruby
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "bento/ubuntu-17.10"
+
+  config.vm.provider "virtualbox" do |vb|
+    vb.cpus   = "4"
+    vb.memory = "4096"
+  end
+
+  config.vm.provision "shell", inline: <<-SHELL
+    #!/bin/sh
+    set -ex
+
+    apt-get update && apt-get install -y apt-transport-https
+    curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+    echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" > /etc/apt/sources.list.d/kubernetes.list
+    apt-get update && apt-get install -y kubelet kubeadm kubectl docker.io
+    
+    swapoff -a
+    systemctl enable docker.service
+
+    kubeadm init
+    export KUBECONFIG=/etc/kubernetes/admin.conf
+    echo "export KUBECONFIG=/etc/kubernetes/admin.conf" >> /etc/bash.bashrc
+    kubectl apply -f https://git.io/weave-kube-1.6
```

Comment:
I'd prefer this:
```
kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
```

ref: https://www.weave.works/docs/net/latest/kubernetes/kube-addon/#install